layouts/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow:
```

Comment:
Do we need to include `/` here to signal robots should not view any page on the site?

Reply:
Ahh, okay. I was under the impression we were disallowing for all, not only ChatGPT. The explicit allowance for all non-ChatGPT makes sense then. Thanks!